src/version.py

```diff
@@ -1,2 +1,6 @@
+"""
+src/version.py
+===========
+"""
```

Comment:
Seriously? You think this module needs docstring?

Reply:
Please look at the [test documentation](https://pybitmessage-test.readthedocs.io/en/latest/index.html) and corresponding PR #1542.